src/Microsoft.Health.Dicom.Core/Features/Common/DicomTagParser.cs

```diff
@@ -14,37 +15,41 @@ namespace Microsoft.Health.Dicom.Core.Features.Common
     /// </summary>
     public class DicomTagParser : IDicomTagParser
     {
-        public bool TryParse(string dicomTagPath, out DicomTag[] dicomTags, bool supportMultiple = false)
+        public bool TryParse(string dicomTagPath, out DicomTag[] dicomTags)
         {
             dicomTags = null;
-            if (supportMultiple)
-            {
-                throw new NotImplementedException(DicomCoreResource.SequentialDicomTagsNotSupported);
-            }
-
             if (string.IsNullOrWhiteSpace(dicomTagPath))
             {
                 return false;
             }
 
+            DicomTag[] tags = dicomTagPath
+                     .Split('.')
+                     .Select(ParseTagFromKeywordOrNumber)
+                     .ToArray();
 
-            DicomTag dicomTag = ParseStardandDicomTagKeyword(dicomTagPath);
-
-            if (dicomTag == null)
+            if (tags.Length > 2)
```

Comment:
Should this be only true for workItem tags or will be get this for Dicom instance tags for free?

Reply:
we get this for Dicom instance tags for free. but we can query for multi level tags in QIDO but it wont return any results with current implementation,

Reply:
You are right, we should block for other than workitem tags. Throwing exception now